framework/doc/content/source/materials/CoupledGradientMaterial.md

```diff
@@ -0,0 +1,9 @@
+# CoupledValueFunctionMaterial / ADCoupledGradientMaterial
+
+!syntax description /Materials/CoupledGradientMaterial
+
```

Comment:
let's add to the stub. ideally you would mention an example, maybe even how you are using it